src/main/java/domain/LottoNumber.java

```diff
@@ -0,0 +1,56 @@
+package domain;
+
+import java.util.Objects;
+import java.util.Random;
+
+public class LottoNumber {
+    public static final int MIN_LOTTO_NUMBER = 1;
+    public static final int MAX_LOTTO_NUMBER = 45;
+    private static final String INPUT_LOTTO_NUMBER_RANGE_OVER_ERROR = "1 ~ 45 사이의 값만 입력하세요!";
+    private static final Random random = new Random();
+
+    private final int lottoNumber;
+
+    public LottoNumber(int lottoNumber) {
+        validateLottoNumberRange(lottoNumber);
+        this.lottoNumber = lottoNumber;
+    }
+
+    private void validateLottoNumberRange(int lottoNumber) {
+        if (isOverRangeOfLottoNumber(lottoNumber)) {
+            throw new IllegalArgumentException(INPUT_LOTTO_NUMBER_RANGE_OVER_ERROR);
+        }
+    }
+
+    private boolean isOverRangeOfLottoNumber(int lottoNumber) {
+        if (MIN_LOTTO_NUMBER > lottoNumber || lottoNumber > MAX_LOTTO_NUMBER) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
return MIN_LOTTO_NUMBER > lottoNumber || lottoNumber > MAX_LOTTO_NUMBER;
```
좀 더 간결하게 사용하실 수 있어요.